lib/search_solr_tools/harvesters/auto_suggest.rb

```diff
@@ -46,8 +46,11 @@ def generate_add_hashes(facet_response, fields)
       end
 
       def add_documents_to_solr(add_docs)
-        if insert_solr_doc add_docs, Base::JSON_CONTENT_TYPE, @env_settings[:auto_suggest_collection_name]
+        status = insert_solr_doc add_docs, Base::JSON_CONTENT_TYPE, @env_settings[:auto_suggest_collection_name]
+
+        if status == Helpers::HarvestStatus::INGEST_OK
```

Comment:
I think you can still do this in one step:
```
if insert_solr_doc add_docs, Base::JSON_CONTENT_TYPE, @env_settings[:auto_suggest_collection_name] == Helpers::HarvestStatus::INGEST_OK
  puts blah blah blah...
```

Reply:
Actually, I don't see a reason to add the step of checking status, since you're still doing a boolean check. It there any reason insert_solr_doc can't continue to return true/false?

Reply:
I probably was overthinking the statuses.  I can look at making changes to have them be integers (though then INGEST_OK would be 0, or false, and the others would be non-zero, or true).  I'll see if I can refactor that a little bit.